app/Http/Controllers/Api/AccessoriesController.php

```diff
@@ -41,10 +41,13 @@ public function index(Request $request)
                 'min_amt',
                 'company_id',
                 'notes',
+                'users_count',
```

Comment:
Does this work? Like, can you sort an accessories list based on the number of checkouts of it?

Reply:
Yes, because of the `->withCount('users' as 'user_count')` - it's already loading those relationships so that's just a quick count on the relationships.